### PR TITLE
Docs: ipmappings

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR adds a documentation page about external host IP remapping.

It also refactors the labels used by the controllers to select the IP resources that need firewall rules inside the gateways.

### CODE REVIEW

File: docs/advanced/external-ip-remapping.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+First of all, export the kubeconfig file of **cluster 1**:
+
+```bash
+export KUBECONFIG=./cluster1-kubeconfig
+```
+
+Get the **configuration** CRD for **cluster 2**:
+
+```bash
+kubectl get configuration -n liqo -o yaml cluster2
```
Comment (fra98) [inline docs/advanced/external-ip-remapping.md:78]: Namespace should be the tenant ns

File: docs/advanced/external-ip-remapping.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+spec:
+...
+status:
+  remote:
+    cidr:
+      external: <REMAPPED_EXT_CIDR>
+      pod: <REMAPPED_POD_CIDR>
+```
+
+Lets focus on the `REMAPPED_EXT_CIDR` value. Keep the *prefix* of that CIDR and replace it inside the `REMAPPED_IP` found in the **IP** CRD status (check the previous section).
```
Comment (fra98) [inline docs/advanced/external-ip-remapping.md:101]:
```suggestion
Let's focus on the `REMAPPED_EXT_CIDR` value. Keep the *prefix* of that CIDR and replace it inside the `REMAPPED_IP` found in the **IP** CRD status (check the previous section).
```

**Review by fra98 — APPROVED** [review]

File: docs/advanced/external-ip-remapping.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
+# External IP remapping
+
+You can use liqo to map external IPs and make them reachable from a peered cluster. You can configure the external IP remapping using the **IP** CRD.
+
+```{warning}
+This feature is available only if [network module](/advanced/manual-peering.md) is enabled.
+```
+
+Check the figure below to understand how the external IP remapping works.
+We are going to make the **external host** reachable from **cluster 1**.
```
Comment (claudiolor) [inline docs/advanced/external-ip-remapping.md:10]: Why cluster 1 needs to pass through cluster 2 to reach the external host? Isn't cluster 1 able to directly reach the host? I believe this is a point that should be clarified

File: docs/advanced/external-ip-remapping.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+kubectl apply -f ip.yaml
+```
+
+Check the status of the **IP** CRD:
+
+```bash
+kubectl get ip external-ip-remap -o yaml
+```
+
+The status should be similar to the following:
```
Comment (claudiolor) [inline docs/advanced/external-ip-remapping.md:52]:
```suggestion
If everything went successfully, the status should be similar to the following:
```

File: docs/advanced/external-ip-remapping.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+## Forge an IP CRD
+
+Export the kubeconfig file of **cluster 2**:
+
+```bash
+export KUBECONFIG=./cluster2-kubeconfig
+```
+
```
Comment (claudiolor) [inline docs/advanced/external-ip-remapping.md:26]: I would clarify here what is the role of the IP CRD

File: docs/advanced/external-ip-remapping.md (hunk 3 above)
Comment (cheina97) [reply docs/advanced/external-ip-remapping.md:10]: Yes, but it's something useless.

**Pull request merged**